app/src/main/res/layout/fragment_dashboard.xml

```diff
@@ -4,7 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingBottom="@dimen/bottom_navigation_height"
+    android:clipToPadding="false"
```

Comment:
Is this attribute entirely necessary in all the places it is in right now? Seems like it's been added in many places.

Reply:
True! I forgot to remove them while trying to figure out what the problem was before 😅